### ISSUE TITLE
cassandra.symmetricEncryptionKey must be configured

### ISSUE THREAD

**Issue by webjoel** [glowroot/glowroot#334]
In Glowroot Central 0.10.4, "Administration", "SMTP configuration", to Save,  occurs error:

```
cassandra.symmetricEncryptionKey must be configured in the glowroot-central.properties file before SMTP password can be saved to cassandra
```

But is configured in properties. I using in container Docker to run cassandra and glowroot central.

**Comment by trask** [comment]
Hi @webjoel, thanks for posting!  Just checking, was `cassandra.symmetricEncryptionKey=` configured in the properties file prior to starting the central collector? (`glowroot-central.properties` is only read at startup)

**Comment by webjoel** [comment]
Yes, I configured. Others options in work's fine, this file

glowroot-central.properties
```
# default is cassandra.contactPoints=127.0.0.1
cassandra.contactPoints=

# default is to connect without credentials
cassandra.username=

# default is to connect without credentials
cassandra.password=

# default is cassandra.keyspace=glowroot
cassandra.keyspace=

# default is cassandra.consistencyLevel=QUORUM
cassandra.consistencyLevel=

cassandra.symmetricEncryptionKey=47b704e87e8a94de976a783d7b68fcb8
...
```

**Comment by trask** [comment]
Hi @webjoel, this is helpful info.  My guess is that the current working directory is not pointing to the `glowroot-central` directory when you are running inside Docker.  I fixed this in the latest snapshot release [glowroot-central-0.10.6-SNAPSHOT-dist.zip](https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=org.glowroot&a=glowroot-central&c=dist&e=zip&v=0.10.6-SNAPSHOT) by getting the `glowroot-central` directory based on the location of the `glowroot-central.jar` file instead of relying on the current working directory to be set correctly.  I think this could also be fixed by setting `WORKDIR` in your `Dockerfile`.  I'm going to look at publishing an official glowroot-central Docker image starting with the next release.

**Comment by webjoel** [comment]
You is correct, I added WORKDIR in Dockerfile and work's fine now. But the fix in 0.10.6 was very important too. Thank's!
